client/packages/system/src/Location/ListView/LocationEditModal/LocationEditModal.tsx

```diff
@@ -150,6 +154,24 @@ export const LocationEditModal: FC<LocationEditModalProps> = ({
             label={t('label.location-type')}
             onChange={locationType => onUpdate({ locationType })}
           />
+          <Box
+            sx={{ display: 'flex', gap: 2, justifyContent: 'space-between' }}
+          >
+            <NumericTextInput
+              value={draft.volume ?? 0}
+              onChange={volume => onUpdate({ volume })}
+              label={t('label.volume')}
+              fullWidth
+              decimalLimit={4}
```

Comment:
to match volume per pack? down below too

```suggestion
              decimalLimit={10}
```